package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack serve --open",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist"
```

Comment:
### Great  job in implementing your package.json
 _Your package.json is implemented correctly ✔️
However the's no need to include the build since you have the 'predeploy'_ 
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d dist"
```
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d dist"
```
```suggestion
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "start": "webpack serve --open",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d dist"
```